TESTING.md

```diff
@@ -32,18 +32,46 @@ describe('src/js/example-web-worker.html', () => {
 ```
 
 And lastly, a test for the React/form/Web worker/WebAssembly combination.
+Let us also change the guess value.
 
 ```{.js file=cypress/integration/example-app_spec.js}
 describe('src/js/example-app.html', () => {
   it('should render -1.00', () => {
     cy.visit('http://localhost:8000/src/js/example-app.html');
-    cy.get('input[name=guess]').type('-30');
+    cy.get('input[name=guess]').type('0');
+    // TODO assert value is set
```

Comment:
Can you make this comment more explicit?

Reply:
Added comment that explains what type does